services/QuillLMS/client/app/bundles/Shared/libs/filterFunctions.tsx

```diff
@@ -0,0 +1,23 @@
+export function filterNumbers(rows, idArray, filterValue) {
+  const id = idArray[0]
+  return rows.filter(row => {
+    let value = filterValue
+    if (value.includes("-")) {
+      let splitStr = filterValue.split("-")
+      if (!isNaN(parseFloat(splitStr[0])) && !isNaN(parseFloat(splitStr[1]))) {
+        return row.original[id] >= splitStr[0] && row.original[id] <= splitStr[1];
+      }
+    } else if (value.includes(">")) {
+      let splitStr = filterValue.split(">")
+      if (!isNaN(parseFloat(splitStr[1]))) {
+        return row.original[id] > splitStr[1]
+      }
+    } else if (value.includes("<")) {
+      let splitStr = filterValue.split("<")
+      if (!isNaN(parseFloat(splitStr[1]))) {
+        return row.original[id] < splitStr[1]
+      }
+    }
+    return true
```

Comment:
I'm having trouble understanding the logic of `return true`.  It seems it could be reached if filterValue had letters or were NaN

Reply:
@brendanshean so I didn't actually write this function, I just pulled it into the Shared bundle. It will be reached if it is letters or NaN, but I think that's what we want-- so that all rows are returned unless triggered with >, < or -

Reply:
Sounds good.  Just wanted to double-check.